python/activity_stream/widget_activity_stream_base.py

```diff
@@ -200,23 +201,11 @@ def _pretty_date(self, time=False):
             return str(day_diff / 30) + " months ago"
         return str(day_diff / 365) + " years ago"
                      
-    # CBB? we are generating HTML links here and they were
-    # styled thusly. i could not style them via setStyleSheet either.
-    # anyway, i think this could be done another way later?
-    # - stewartb
     def __generate_url(self, entity_type, entity_id, name):
         """
         Generate a standard shotgun url
         """
-        str_val = """<a href='%s:%s' 
-                        style='text-decoration: none;
-                        color: %s'>%s</a>
-                  """ % (entity_type, 
-                         entity_id,
-                         #self._bundle.style_constants["SG_HIGHLIGHT_COLOR"],
-                         'rgb(126,127,129);',
-                         name)
-        return str_val
+        return get_hyperlink_html("{0}:{1}".format(entity_type, entity_id), name)
```

Comment:
Same comments as other PR regarding python version and explicit replacement names. The latter just being a personal preference.